packages/manager/apps/hycu/src/components/Order/PackSelection.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Trans, useTranslation } from 'react-i18next';
+
+import { OdsHTMLAnchorElementTarget } from '@ovhcloud/ods-common-core';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  ODS_BUTTON_VARIANT,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import {
+  OsdsText,
+  OsdsButton,
+  OsdsTile,
+  OsdsLink,
+} from '@ovhcloud/ods-components/react';
+import {
+  Description,
+  OvhSubsidiary,
+  Price,
+  Subtitle,
+} from '@ovh-ux/manager-react-components';
+
+import Loading from '@/components/Loading/Loading.component';
+import { useOrderCatalogHYCU } from '@/hooks/order/useOrderCatalogHYCU';
+import { urls } from '@/routes/routes.constant';
+import { sortPacksByPrice } from '@/utils/sortPacks';
+import { getRenewPrice } from '@/utils/getRenewPrice';
+
+type PackSelectionProps = {
+  selectPack: Dispatch<SetStateAction<string>>;
+  setOrderInitiated: () => void;
+  orderLink: string;
+  selectedPack: string;
+  subsidiary: OvhSubsidiary;
+};
+
+const PackSelection = ({
+  selectPack,
+  setOrderInitiated,
+  orderLink,
+  selectedPack,
+  subsidiary,
+}: PackSelectionProps) => {
+  const { t } = useTranslation('order');
+  const navigate = useNavigate();
+
+  const { data: orderCatalogHYCU, isLoading } = useOrderCatalogHYCU(subsidiary);
+
+  return (
+    <>
+      <Subtitle className="block mb-6">{t('hycu_order_subtitle')}</Subtitle>
+      <Description className="mb-8">
+        <Trans
+          t={t}
+          i18nKey="hycu_order_subtitle_description"
+          components={{
+            contact: (
+              <OsdsLink
+                color={ODS_THEME_COLOR_INTENT.primary}
+                href={'https://www.ovhcloud.com/fr/contact/'}
```

Comment:
I think it should be a constant and it will probably depends on the subsidiary (like the onboarding guides) so you will probably need to create an object with all the variants and get the right link with the subsidiary